tests-functional/tests/test_backup_mnemonic_and_restore.py

```diff
@@ -0,0 +1,187 @@
+import random
+import string
+import threading
+from clients.status_backend import StatusBackend
+import pytest
+
+from resources.constants import user_mnemonic_12, user_mnemonic_15, user_mnemonic_24
+from resources.utils import assert_response_attributes
+
+
+@pytest.mark.create_account
+@pytest.mark.rpc
+class TestBackupMnemonicAndRestore:
+
+    @pytest.fixture(autouse=True)
+    def setup_cleanup(self, close_status_backend_containers):
+        """Automatically cleanup containers after each test"""
+        yield
+
+    def test_profile_creation_and_mnemonics_backup(self):
+        # Create a new account container and initialize
+        account = StatusBackend()
+        account.init_status_backend()
+        account.create_account_and_login()
```

Comment:
`self.wait_for_login` is required after logging in. Otherwise the setting might not have been yet initialized.